src/main/io/osd.c

```diff
@@ -180,305 +183,275 @@ static void osdDrawSingleElement(uint8_t item)
     char buff[32];
 
     switch(item) {
-        case OSD_RSSI_VALUE:
-        {
-            uint16_t osdRssi = rssi * 100 / 1024; // change range
-            if (osdRssi >= 100)
-                osdRssi = 99;
-
-            buff[0] = SYM_RSSI;
-            tfp_sprintf(buff + 1, "%d", osdRssi);
-            break;
-        }
+    case OSD_RSSI_VALUE:
+    {
```

Comment:
The agreed standard for `switch` statements is that `case` is aligned with `switch` and code is indented inside `case` statements, this includes braces, if required. So if braces are required we have:
```
switch (item) {
case x:
    {
        statement;
    }
case y:
...
```